A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/Clock/A320_Neo_Clock.js

```diff
@@ -78,25 +96,36 @@ class A320_Neo_Clock extends BaseAirliners {
         if (value) {
             const seconds = Number.parseInt(value);
             const time = Utils.SecondsToDisplayTime(seconds, true, true, false);
-            return time.toString();
+            return time.toString().substr(0,5);
         }
         return "";
     }
 
+    getUTCSeconds() {
+        const value = SimVar.GetGlobalVarValue("ZULU TIME", "seconds");
+        if (value) {
+            const seconds = Number.parseInt(value);
+            const time = Utils.SecondsToDisplayTime(seconds, true, true, false);
+            return time.toString().substr(6,2);
+        }
+        return "";
+    }
     getUTCDate() {
-        const Day = SimVar.GetGlobalVarValue("ZULU DAY OF MONTH", "number")
-        const Month = SimVar.GetGlobalVarValue("ZULU MONTH OF YEAR", "number")
-        const Year = `${SimVar.GetGlobalVarValue("ZULU YEAR", "number")}`.substr(2,4)
-
-        return `${Day}.${Month}.${Year}`
+        const Day = SimVar.GetGlobalVarValue("ZULU DAY OF MONTH", "number");
+        const Month = `${SimVar.GetGlobalVarValue("ZULU MONTH OF YEAR", "number")}`.padStart(2,'0');
+        return `${Month}.${Day}`;
     }
 
+    getUTCYear() {
+        const Year = `${SimVar.GetGlobalVarValue("ZULU YEAR", "number")}`.substr(2,4);
+        return `${Year}`;
```

Comment:
```suggestion
        const year = `${SimVar.GetGlobalVarValue("ZULU YEAR", "number")}`.substr(2,4);
        return `${year}`;
```

Reply:
this var was labelled like this in the original ASOBO code